+++
title = "Colliding Galaxies, Machine Learning and Systems Medicine: Could Machine Learning Ever Cure Alzheimer's Disease?"
authors = ["Prof Winston Hide, University of Sheffield"]
date = 2017-10-25T12:56:39Z  # Schedule page publish date.
draft = false

# Talk start and end times
time_start = 2017-10-25T15:30:00Z
time_end = 2017-10-25T16:30:00Z

# Optional shortened abstract
abstract="" # ignore this
abstract_short = ""

# Name of event and optional event URL.
event = "Machine Learning Seminar"
event_url = ""

# Location of event.
location = "Ada Lovelace (Regent Court COM-108)"

# Seminar host
host="Haiping Lu"

# Is this a selected talk? (true/false)
selected = false

tags = ["seminar"]

# Links (optional).
url_pdf = ""
url_slides = ""
url_video = ""
url_code = ""

# Does the content use math formatting?
math = false

# Does the content use source code highlighting?
highlight = true

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = ""
caption = ""

+++

## Abstract

Recent advances in application of machine learning to genomic scale data interpretation for drug repurposing have brought biotechnology and data science into stark relief. This sea change reflects the challenge and opportunities created by two behemoths which like galaxies, are slowly colliding. Organisations are waking up to the value of having information that comes from existing datasets, generating targeted data, and looking within it to drive insight, rather than establishing an hypothesis about a particular drug target/drug combination and finding anecdotal data to support or refute it.

As deep learning is more broadly applied it has started to touch the world of this high dimensional data driven drug discovery. But the application of drug repurposing to systems medicine rests to a large extent on better understanding of the pharmacological properties of small molecules based on their transcriptional response signatures in cell lines and model systems.

To move the field forwards, ML and Systems Medicine approaches can be combined to leverage the ability of ML to learn and predict across large data while exploiting systems aspects of networked relationships of genes and how the topology of these relationships drive disease. Both must overcome the diversity and noise of data types, the challenge of data sharing, and a better understanding of the underlying features which drive selection of drug and target combinations. Both must have far better benchmarking to drive definition of well defined outcomes.

The presentation will describe the collision points with machine learning and the steps we are taking to help address its inherent challenges in successful prediction of drug/target combinations.

## Biography

Winston Hide gained a PhD at Temple University in Philadelphia in 1992, and performed post-doctoral training in molecular evolution at the University of Texas in Houston, Baylor College of Medicine at Human Genome Centre with Richard Gibbs and at the Smithsonian Museum of Natural History in Washington DC. He was as director of genomics at MasPar high performance computer corporation in Silicon Valley, then he returned to South Africa to found and direct the South African National Bioinformatics Institute (SANBI) at the University of Western Cape in 1996. An author of the National Biotechnology Strategy for South Africa he co-founded the South African National Bioinformatics Network. A Kerr Fellow of the Ludwig Cancer Institute, he was a member of the Brazilian team investigating the cancer transcriptome. A co-founder of the WHO regional training programme in Bioinformatics, he co-led development of training for bioinformatics across the Southern Hemisphere. Focusing on the development of Africa's peoples he is a founder member of the steering committee that established the NIH-Wellcome Trust funded Pan African H3 Africa genome Initiative. Together with a group from the WHO, Yale, and the US Sanger Center he established the International Glossina Genome Initiative in 2005 which culminated in the publication of the Tsetse Fly genome in 2014. Hide has been recognised for these activities by receipt of the first International Society for Computational Biology award for outstanding achievement.

Hide was until recently, Associate Professor of Bioinformatics and Computational Biology in the Department of Biostatistics at Harvard School of Public Health where he has led development of bioinformatics addressing personal genomics approaches to public health and where he directed the HSPH Bioinformatics Core. Hide developed the bioinformatics strategy for the Harvard Stem Cell Institute and was Director of its Center for Stem Cell Bioinformatics where he built a science commons for big data sharing and integration.

Hide now is Chair in Computational Biology at the University of Sheffield where he drives development of systems medicine and genome translation at the Sheffield Institute for Translational Neurosciences.

His research addresses integration of genomic data to deliver clinical translation of genomics for application to repurposing of drugs, to determine prioritised drug targets and to deliver prediction to predisposition to disease. Hide directs the bioinformatics for the Alzheimer's Genome Project (Cure Alzhiemer's Foundation) where he has built the computational infrastructure for, and analyzed 1500 whole genome sequences from patients with the disease and is a driving member of the CureAD CIRCUITS consortium, a group made up of scientists from Harvard, MIT, USCF and Sheffield funded by the Cure Alzhiemer's Foundation to determine the regulatory processes that go awry in Alzheimer's. Currently a visiting faculty member of the Computer Science and Artificial Intelligence Laboratory at MIT, Hide also leads the Sheffield Node of the Connected Health Cities Consortium of Northern Universities.
