+++
title = "Bayesian modelling of allelic expression for somatic clones in single cells"
authors = ["Yuanhua Huang, European Bioinformatics Institute - EMBL-EBI"]
date = 2019-10-30T21:00:00Z  # Schedule page publish date.
draft = false

# Talk start and end times
time_start = 2019-12-04T15:00:00Z
time_end = 2019-12-04T16:00:00Z

# Optional shortened abstract
abstract="" # ignore this
abstract_short = ""

# Name of event and optional event URL.
event = "Machine Learning Seminar"
event_url = ""

# Location of event.
location = "COM-108 Ada Lovelace Room (Regent Court), DCS"

# Seminar host
host="Mauricio Alvarez"

# Is this a selected talk? (true/false)
selected = false

tags = ["seminar"]

# Links (optional).
url_pdf = ""
url_slides = ""
url_video = ""
url_code = ""

# Does the content use math formatting?
math = false

# Does the content use source code highlighting?
highlight = true

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = ""
caption = ""

+++

## Abstract

Decoding the clonal substructures of somatic tissues sheds light on ageing and disease. However, approaches to systematically characterize phenotypic and functional variations between individual clones are not established. Here we will present cardelino, a Bayesian method for inferring the clonal mutations and the identity of individual cells by modelling the expressed variants in single-cell RNA-seq (scRNA-seq) data. Critically, cardelino allows effective integration of additional but often noisy clonal tree, e.g., from bulk exome-seq data. We applied cardelino to 32 human dermal fibroblast lines, identifying hundreds of differentially expressed genes between clones, which are frequently enriched in cell cycle and proliferation pathways.

## Biography

Yuanhua Huang is currently an EBPOD postdoctoral fellow jointly between the University of Cambridge and European Bioinformatics Institute. He obtained a PhD in informatics from the University of Edinburgh in 2017. His research focuses on developing Bayesian machine learning methods to study biological questions, mainly by modelling genomic sequencing data. Besides genetic variations mentioned in this seminar, he also developed a set of Bayesian models to account for latent structures in quantification of RNA splicing, e.g., Gaussian processes for modelling temporal structure in time-series RNA-seq data.